### PR TITLE
ci: add windows

### PR DESCRIPTION
**Pull request by jrandolf**
Bug: #876

### CODE REVIEW

File: tools/run-wpt.mjs [hunk 1]
```diff
@@ -43,17 +45,35 @@ if (
...
-  (isLinux
-    ? '/usr/bin/google-chrome-unstable'
-    : isMacOS
-    ? '/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev'
-    : '');
+let BROWSER_BIN = process.env.BROWSER_BIN;
+if (!BROWSER_BIN) {
+  switch (process.platform) {
+    case 'linux':
+      BROWSER_BIN = join(sep, 'usr', 'bin', 'google-chrome-unstable');
```
Comment (thiagowfx) [inline tools/run-wpt.mjs:52]: More readable, no?

      BROWSER_BIN = '/usr/bin/google-chrome-unstable';

Comment (thiagowfx) [reply tools/run-wpt.mjs:52]: Same for macOS.

Comment (thiagowfx) [reply tools/run-wpt.mjs:52]: It's just odd to use both `join` and `sep` at the same time. The fact you put `sep` at the beginning paramounts to hard-coding the fact Unix absolute paths start with `/`. So you might as well completely remove `sep`.

**Review by thiagowfx — APPROVED** [review]

File: tools/run-wpt.mjs (hunk 1 above)
Comment (jrandolf) [reply tools/run-wpt.mjs:52]: This is more programmatic which is why I chose it. Since this is more versatile, I'll just keep it as it.

**Comment by thiagowfx** [comment]
Did you force-merge? Windows E2E is failing.

**Comment by jrandolf** [comment]
Windows E2E is not required so it merged.

**Comment by thiagowfx** [comment]
If they do not work, it's better to remove them for now. Even if they are non-blocking, they make the overall PR status be "X" (fail / red), which is misleading.

**Pull request merged**